lib/App/DH.pm

```diff
@@ -261,15 +261,31 @@ sub cmd_upgrade {
   return;
 }
 
+=cmd version
+
+Report version of connection L</--connection_name>
+
+    dh.pl [...params] version
+
+=cut
+
+sub cmd_version {
+  my $self = shift;
+  print $self->_dh->schema_version . "\n";
```

Comment:
Perl critic is complaining that this line is "un-checked", that is, the print could fail needing action. But I don't think any action is required here on failure, so a 

```perl
## no critic (RequireCheckedSyscalls)
```

Should do the trick.